docs/python_binding.rst

```diff
@@ -107,6 +107,14 @@ To get a python dictionary of the settings, call ``session::get_settings``.
 
 .. _`library reference`: reference.html
 
+
+Retrieving session statistics in Python is more convenient than that in C++.
+
+In C++, one should call ``post_session_stats()`` in the ``session`` object to get ``session_stats_alert``, in which the statistics is stored as an array.
```

Comment:
this gives the impression that calling ``post_session_stats()`` is a C++ thing (I think). Would it improve things to move the "in C++" to the next sentence?